src/network/bmproto.py

```diff
@@ -575,3 +576,32 @@ def handle_close(self):
             except AttributeError:
                 logger.debug("Disconnected socket closing")
         AdvancedDispatcher.handle_close(self)
+
+
+class BMStringParser(BMProto):
+    """
+    A special case of BMProto used by objectProcessor to send ACK
+    """
+    def __init__(self):
+        super(BMStringParser, self).__init__()
+        self.destination = state.Peer('127.0.0.1', 8444)
+        self.payload = None
+        ObjectTracker.__init__(self)
+
+    def send_data(self, data):
+        """Send object given by the data string"""
+        # This class is introduced specially for ACK sending, please
+        # change log strings if you are going to use it for something else
+        self.bm_proto_reset()
+        self.payload = data
+        try:
+            self.bm_command_object()
+        except BMObjectAlreadyHaveError:
+            pass  # maybe the same msg received on different nodes
+        except BMObjectExpiredError:
+            logger.debug(
+                'Sending ACK failure (expired): %s', hexlify(data))
+        except Exception as e:
```

Comment:
`BMObjectUnwantedStreamError` probably needs additional coding if the creator of the ACK isn't in of of my streams. Which could happen in the future. But this should probably be addressed in `bm_command_object` and for the time being doesn't matter. It will be sufficiently tested once streams are implemented.